seafile/Base.lproj/SeafAccountCell.xib

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is already SeafAccountCell.xib and SeafCell.xib in en.lproj, why create another under Base.lproj?